### PR TITLE
Reset option value to default for privacy policy on site empty

### PR DESCRIPTION
**Pull request by thrijith**
Fixes https://github.com/wp-cli/wp-cli/issues/5086

### CODE REVIEW

**Comment by thrijith** [comment]
Hi @danielbachhuber, I have added a test in an existing scenario. LMK if it's ok.

File: features/site-empty.feature [hunk 1]
```diff
@@ -42,6 +42,12 @@ Feature: Empty a WordPress site of its data
     When I run `wp term list post_tag --format=ids`
     Then STDOUT should be empty
 
+    When I run `wp option get wp_page_for_privacy_policy`
+    Then STDOUT should be:
+      """
+      0
+      """
```
Comment (danielbachhuber) [inline features/site-empty.feature]: Can we also include a test that makes sure there's a non-zero value _before_ `wp site empty` is run?

Comment (thrijith) [reply features/site-empty.feature]: I am not sure how to do that, should I create a dummy page and assign it as privacy page?

Comment (danielbachhuber) [reply features/site-empty.feature]: > should I create a dummy page and assign it as privacy page?

Yes, this would be fine.
